.github/workflows/latest-matrix-sdk-crypto.yml

```diff
@@ -0,0 +1,87 @@
+name: Build against latest matrix-sdk-crypto
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '22 1 * * *'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  CARGO_TERM_COLOR: always
+
+permissions:
+  contents: read
+  issues: write
+
+jobs:
+  test-against-latest-matrix-sdk-crypto:
+    name: test-against-latest-matrix-sdk-crypto
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@v3
+
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          targets: wasm32-unknown-unknown
+
+      - name: Load cache
+        uses: Swatinem/rust-cache@v2
+        with:
+          save-if: ${{ github.ref == 'refs/heads/main' }}
+
+      - name: Update to the latest matrix-sdk-crypto version
+        run: cargo update matrix-sdk-crypto
+
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20.0
+
+      - name: Install yarn
+        run: npm install --global yarn
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Build the WebAssembly + JavaScript binding
+        run: yarn build
+
+      - name: Test the JavaScript binding
+        run: yarn test
+
+      - name: Open an issue on failure
+        if: failure()
+        uses: JasonEtco/create-an-issue@1b14a70e4d8dc185e5cc76d3bec9eab20257b2c5 # v2.9.2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          update_existing: true
+          filename: .github/latest_matrix_sdk_failed_issue_template.md
+        id: create-issue
+
+      - run: 'echo Created/updated ${{ steps.create-issue.outputs.url }}'
+        if: failure()
+
+      - name: Comment on the issue
+        if: failure()
+        uses: thollander/actions-comment-pull-request@v3
+        with:
+          message: |
+            Building matrix-rust-sdk-crypto-wasm against the latest matrix-sdk Rust is failing.
+            See https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
+          pr-number: "${{ steps.create-issue.outputs.number }}"
```

Comment:
Does it mean that the first time, it will create both the issue *and* comment on it? And subsequent times, it will only comment?

(As an intuition: it might be a good thing to explain at the top of this yaml file, in a large comment, what it does overall, and how it's supposed to behave, so one doesn't have to read the entire source, maybe? With the risk that it could be desync'd from the code, indeed…)

Reply:
f5ec776